jupyter_myst_build_proxy/remixContextMod.html

```diff
@@ -0,0 +1,10 @@
+<script>
+  rebuildAction = {
+    title: "Rebuild",
+    url: `${window.location.pathname}?rebuild=1`,
+    internal: true,
+    static: true,
+  };
+  window.__remixContext["state"]["loaderData"]["root"]["config"]["actions"].unshift(rebuildAction);
+  window.__remixContext["state"]["loaderData"]["routes/$"]["config"]["actions"].unshift(rebuildAction);
```

Comment:
I think I'm going to change this to append instead of prepend so we can more clearly describe the location of the rebuild button at the top-right in a classroom scenario.